app/src/common/chromium/com/igalia/wolvic/browser/api/impl/TabWebContentsDelegate.java

```diff
@@ -16,8 +19,14 @@
 import org.chromium.url.GURL;
 import org.chromium.wolvic.Tab;
 import org.chromium.wolvic.WolvicWebContentsDelegate;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.io.IOException;
 
 public class TabWebContentsDelegate extends WolvicWebContentsDelegate {
+    private static final String TAG = "TabWebContentsDelegate";
```

Comment:
In Wolvic we use LOGTAG instead. And we create it this way
`private static final String LOGTAG = SystemUtils.createLogtag(TabWebContentsDelegate.class);`

Reply:
Done